### PR TITLE
restructured and improved naming for anconda package

### PR DESCRIPTION
**Pull request by Alaukiksaxena**
Restructured and added some new functionalities. Corrected some bugs. Some other functionalities will follow. However, the main structure of the code and the main classes will remain the same.

### CODE REVIEW

**Comment by srmnitc** [comment]
@Alaukiksaxena The tests are failing, this is because of the changed function names. I am not sure if this is something you wanted or not.

File: compositionspace/datautils.py [hunk 1]
```diff
@@ -222,29 +201,42 @@ def read_apt_to_df(self):
...
+            if filename.endswith(".RRNG"):
+                path = os.path.join(self.params["input_path"], filename) 
+                ions,rrngs = self.get_rrng(rrange_file)
                 
         return (df_Mass_POS_lst, file_name_lst, ions, rrngs) 
 
 
 
-    def chunkify_apt_df(self):
+    def get_big_slices(self):
```
Comment (srmnitc) [inline compositionspace/datautils.py:239]: I am completely fine with function name changes, just please also update in the tests (in the tests folder), and it should be fine.

File: compositionspace/segmentation.py [hunk 2]
```diff
@@ -30,7 +30,7 @@ def get_PCA_cumsum(self, vox_ratio_file, vox_file):
         with h5py.File(vox_file,"r") as hdf:
             group = hdf.get("Group_sm_vox_xyz_Da_spec")
             group0 = hdf.get("0")
-            spec_lst = list(list(group0.attrs.values())[2])
+            spec_lst = list(list(group0.attrs.values())[1])
```
Comment (srmnitc) [inline compositionspace/segmentation.py:33]: Is this correct, I remember that index 1 did not give me the right values, which is why I changed it.

File: tests/experiment_params.yaml [hunk 3]
```diff
@@ -1,18 +1,18 @@
-input_path: tests/data
+input_path: data
```
Comment (srmnitc) [inline tests/experiment_params.yaml:1]: This should be put back to `tests/data` for the tests to run properly

**Review by srmnitc — CHANGES_REQUESTED** [review]
The tests are failing because the function names have been changes, but not updated in the tests. Once you fix this, we can go ahead and merge it.

File: compositionspace/segmentation.py (hunk 2 above)
Comment (Alaukiksaxena) [reply compositionspace/segmentation.py:33]: Actually, you might remember we got some weird results in the meeting a few days ago. This was the reason.

File: tests/experiment_params.yaml (hunk 3 above)
Comment (Alaukiksaxena) [reply tests/experiment_params.yaml:1]: I will change this

File: compositionspace/datautils.py (hunk 1 above)
Comment (Alaukiksaxena) [reply compositionspace/datautils.py:239]: I will change this

File: compositionspace/segmentation.py (hunk 2 above)
Comment (srmnitc) [reply compositionspace/segmentation.py:33]: Perfect, then this is good.

File: compositionspace/datautils.py (hunk 1 above)
Comment (srmnitc) [reply compositionspace/datautils.py:239]: This is also true for a number of other functions, you could either put back the names; or change in tests, whichever you prefer. From my side, both are ok.

**Comment by Alaukiksaxena** [comment]
Done from my side

**Comment by srmnitc** [comment]
The tests seems to pass, thanks a lot!

**Pull request merged**